fastapi/utils.py

```diff
@@ -93,9 +93,7 @@ def create_cloned_field(field: ModelField) -> ModelField:
     use_type = original_type
     if lenient_issubclass(original_type, BaseModel):
         original_type = cast(Type[BaseModel], original_type)
-        use_type = create_model(
-            original_type.__name__, __config__=original_type.__config__
-        )
+        use_type = create_model(original_type.__name__, __base__=original_type)
```

Comment:
This change looks correct to me, based on the [current v1.4 definition](https://github.com/samuelcolvin/pydantic/blob/v1.4/pydantic/main.py#L736), and old [v0.32 definition](https://github.com/samuelcolvin/pydantic/blob/v0.32.2/pydantic/main.py#L657).